mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -243,6 +250,34 @@ impl PlatformCoinWithTokensActivationOps for TendermintCoin {
                 });
             }
 
+            // Send migration request
+            #[cfg(not(target_arch = "wasm32"))]
+            {
+                let db_id = hex::encode(dhash160(pubkey.to_bytes().as_slice()));
+                let shared_db_id = shared_db_id_from_seed(&pubkey.to_hex())
+                    .mm_err(|err| TendermintInitError {
+                        ticker: ticker.clone(),
+                        kind: TendermintInitErrorKind::Internal(err.to_string()),
+                    })?
+                    .to_string();
+
+                let db_migration_sender = ctx.db_migration_watcher().await.get_sender().await;
+                let mut db_migration_sender = db_migration_sender.lock().await;
+                db_migration_sender
+                    .send(DbIds {
+                        db_id: db_id.clone(),
+                        shared_db_id: shared_db_id.clone(),
+                    })
+                    .await
+                    .map_to_mm(|err| TendermintInitError {
+                        ticker: ticker.clone(),
+                        kind: TendermintInitErrorKind::Internal(err.to_string()),
+                    })?;
+
+                info!("Public key hash: {db_id}");
+                info!("Shared Database ID: {shared_db_id}");
```

Comment:
These are useful only when debugging

```suggestion
                debug!("Public key hash: {db_id}");
                debug!("Shared Database ID: {shared_db_id}");
```